src/main/java/com/criteo/nosql/mewpoke/couchbase/CouchbaseMonitor.java

```diff
@@ -113,17 +113,19 @@ private static InetSocketAddress nodeToInetAddress(Node n) {
         return new InetSocketAddress(n.hostname().hostname(), 8091);
     }
 
-    public static Optional<CouchbaseMonitor> fromNodes(final Service service, Set<InetSocketAddress> endPoints, long timeoutInMs, String username, String password, Config.CouchbaseStats bucketStats) {
+    public static Optional<CouchbaseMonitor> fromNodes(final Service service, Set<InetSocketAddress> endPoints, long timeoutInMs, String username, String password, Config config) {
```

Comment:
It is strange to manage differently bucketpassword and timeoutInMs/username/password. So OR we pass only Config in parameter, OR we pass bucketpassword as String